[28/Jun/2016 20:35:36] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1344, in can_remove
    form_data = FormData.objects.get(id=form_data_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormData matching query does not exist.
[28/Jun/2016 20:35:44] ERROR [django.request:231] Internal Server Error: /curate/generate_xsd_form
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 2051, in generate_xsd_form
    formString += generateForm(request)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1935, in generateForm
    form_data = FormData.objects.get(pk=ObjectId(form_data_id))
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormData matching query does not exist.
[29/Jun/2016 14:13:29] WARNING [django.request:143] Not Found: /favicon.ico
[29/Jun/2016 15:34:30] WARNING [django.request:143] Not Found: /favicon.ico
[29/Jun/2016 15:42:43] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dd875d0>.
[29/Jun/2016 15:43:54] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78cf31450>.
[29/Jun/2016 15:44:00] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78cf57d50>.
[29/Jun/2016 15:44:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dd90e10>.
[29/Jun/2016 15:44:22] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78d8f93d0>.
[29/Jun/2016 15:45:29] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dd87210>.
[29/Jun/2016 15:45:33] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7748d7390>.
[29/Jun/2016 15:45:49] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7729b3050>.
[29/Jun/2016 15:45:58] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7725fc310>.
[29/Jun/2016 15:46:47] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7729b3e10>.
[29/Jun/2016 15:48:08] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dc6a610>.
[29/Jun/2016 15:48:17] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dc6ac90>.
[29/Jun/2016 15:48:17] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dc6a790>.
[29/Jun/2016 15:48:30] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd748693b10>.
[29/Jun/2016 15:48:30] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7449be350>.
[29/Jun/2016 15:48:49] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6cc0d0>.
[29/Jun/2016 15:48:49] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dc6a310>.
[29/Jun/2016 15:48:57] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dc6a650>.
[29/Jun/2016 15:48:57] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c18e2d0>.
[29/Jun/2016 15:48:59] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b698d0>.
[29/Jun/2016 15:48:59] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c18e3d0>.
[29/Jun/2016 15:49:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd760dc47d0>.
[29/Jun/2016 15:49:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7725f1290>.
[29/Jun/2016 15:51:32] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747fa910>.
[29/Jun/2016 15:51:36] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747fa6d0>.
[29/Jun/2016 15:51:40] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd748693750>.
[29/Jun/2016 15:51:48] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd74583b5d0>.
[29/Jun/2016 15:51:51] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747f59d0>.
[29/Jun/2016 15:52:00] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747fac90>.
[29/Jun/2016 15:52:39] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5550>.
[29/Jun/2016 15:56:18] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd74583b950>.
[29/Jun/2016 15:56:29] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5d50>.
[29/Jun/2016 15:59:58] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7484ce950>.
[29/Jun/2016 16:00:00] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b7fd0>.
[29/Jun/2016 16:00:05] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b7f10>.
[29/Jun/2016 16:00:06] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7c69d0>.
[29/Jun/2016 16:01:28] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7ba8d0>.
[29/Jun/2016 16:01:30] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747cab90>.
[29/Jun/2016 16:01:34] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7c6ed0>.
[29/Jun/2016 16:04:54] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7b7cd0>.
[29/Jun/2016 16:11:12] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd748693410>.
[29/Jun/2016 16:12:53] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c545410>.
[29/Jun/2016 16:12:58] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c18ee10>.
[29/Jun/2016 16:13:09] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747dcd10>.
[29/Jun/2016 16:13:14] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd74583bd10>.
[29/Jun/2016 16:13:15] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7748d79d0>.
[29/Jun/2016 16:13:17] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c18e410>.
[29/Jun/2016 16:13:19] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7748d71d0>.
[29/Jun/2016 16:13:26] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747ca0d0>.
[29/Jun/2016 16:13:27] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7486932d0>.
[29/Jun/2016 16:13:29] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7486930d0>.
[29/Jun/2016 16:13:31] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd748693fd0>.
[29/Jun/2016 16:13:49] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6c6290>.
[29/Jun/2016 16:13:51] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dc6abd0>.
[29/Jun/2016 16:13:53] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747cab90>.
[29/Jun/2016 16:13:57] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6cc4d0>.
[29/Jun/2016 16:14:13] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7449be1d0>.
[29/Jun/2016 16:14:16] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dc6a090>.
[29/Jun/2016 16:14:35] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd748693dd0>.
[29/Jun/2016 16:14:37] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7484b4f50>.
[29/Jun/2016 16:14:41] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c4bedd0>.
[29/Jun/2016 16:14:43] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b7190>.
[29/Jun/2016 16:50:54] WARNING [django.request:143] Not Found: /favicon.ico
[29/Jun/2016 18:14:14] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5990>.
[29/Jun/2016 18:14:16] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5a50>.
[29/Jun/2016 18:14:21] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e51d0>.
[29/Jun/2016 18:14:32] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7b7a90>.
[29/Jun/2016 18:14:42] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b70d0>.
[29/Jun/2016 18:14:46] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7729b3f50>.
[29/Jun/2016 18:14:51] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7449ccad0>.
[29/Jun/2016 18:15:03] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747cab10>.
[29/Jun/2016 18:15:06] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6c6690>.
[29/Jun/2016 18:15:06] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6c6b90>.
[29/Jun/2016 18:15:08] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7bafd0>.
[29/Jun/2016 18:15:08] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd760dc46d0>.
[29/Jun/2016 18:15:12] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747caf90>.
[29/Jun/2016 18:15:12] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6c6e90>.
[29/Jun/2016 18:15:18] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747ca4d0>.
[29/Jun/2016 18:15:18] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b7650>.
[29/Jun/2016 18:15:34] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e59d0>.
[29/Jun/2016 18:15:34] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7bafd0>.
[29/Jun/2016 18:21:47] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b54050>.
[29/Jun/2016 18:21:47] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b54f90>.
[29/Jun/2016 18:21:59] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b54e90>.
[29/Jun/2016 18:21:59] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78cf57290>.
[29/Jun/2016 18:22:05] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7449cc6d0>.
[29/Jun/2016 18:22:05] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5cd0>.
[29/Jun/2016 18:22:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78cf57b90>.
[29/Jun/2016 18:22:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5450>.
[29/Jun/2016 18:22:11] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5450>.
[29/Jun/2016 18:22:11] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c164c50>.
[29/Jun/2016 18:25:57] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5710>.
[29/Jun/2016 18:25:57] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b69710>.
[29/Jun/2016 18:26:02] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5710>.
[29/Jun/2016 18:26:02] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b69d10>.
[29/Jun/2016 18:26:03] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7725f1cd0>.
[29/Jun/2016 18:26:03] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dd90e50>.
[29/Jun/2016 18:26:15] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c189e50>.
[29/Jun/2016 18:26:15] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b69d10>.
[29/Jun/2016 18:26:17] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b69e10>.
[29/Jun/2016 18:26:17] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dd90e50>.
[29/Jun/2016 18:26:20] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd760dc4ed0>.
[29/Jun/2016 18:26:20] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b69c90>.
[29/Jun/2016 18:26:29] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747cab10>.
[29/Jun/2016 18:26:29] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747f57d0>.
[29/Jun/2016 18:26:38] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b75d0>.
[29/Jun/2016 18:26:38] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5410>.
[29/Jun/2016 18:26:44] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b7f10>.
[29/Jun/2016 18:26:44] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b69250>.
[29/Jun/2016 18:26:48] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd771b69450>.
[29/Jun/2016 18:26:48] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b7f10>.
[29/Jun/2016 18:26:50] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7449cce50>.
[29/Jun/2016 18:26:50] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x28dcb50>.
[29/Jun/2016 18:27:00] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7729b37d0>.
[29/Jun/2016 18:27:00] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7729b37d0>.
[29/Jun/2016 18:27:04] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5c90>.
[29/Jun/2016 18:27:04] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7484ce5d0>.
[29/Jun/2016 18:27:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6c6a50>.
[29/Jun/2016 18:27:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd772d2a350>.
[29/Jun/2016 18:27:09] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747e5ad0>.
[29/Jun/2016 18:27:09] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747b7f90>.
[29/Jun/2016 18:37:35] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c7b7490>.
[29/Jun/2016 18:37:54] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7747eef50>.
[29/Jun/2016 18:38:02] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78c6c5150>.
[29/Jun/2016 18:38:05] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd77c18ee10>.
[29/Jun/2016 18:38:51] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78dd87050>.
[29/Jun/2016 18:39:09] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7484b4a90>.
[29/Jun/2016 21:13:11] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:12] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:13] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:13] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:16] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:18] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:21] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:22] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:25] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:27] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:29] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:29] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:31] ERROR [django.request:231] Internal Server Error: /curate/remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1381, in remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:31] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:32] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:33] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:35] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:36] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:37] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:39] ERROR [django.request:231] Internal Server Error: /curate/remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1381, in remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:39] ERROR [django.request:231] Internal Server Error: /curate/remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1381, in remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:39] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:40] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1346, in can_remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[29/Jun/2016 21:13:41] ERROR [django.request:231] Internal Server Error: /curate/remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1381, in remove
    form_element = FormElement.objects.get(id=form_element_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormElement matching query does not exist.
[30/Jun/2016 00:39:59] WARNING [django.request:143] Not Found: /robots.txt
[30/Jun/2016 02:57:00] WARNING [django.request:143] Not Found: /http://www.baidu.com/cache/global/img/gs.gif
[30/Jun/2016 03:58:15] WARNING [django.request:143] Not Found: /favicon.ico
[30/Jun/2016 14:50:01] WARNING [django.request:143] Not Found: /favicon.ico
[30/Jun/2016 14:55:10] WARNING [django.request:143] Not Found: /favicon.ico
[30/Jun/2016 14:55:28] WARNING [django.request:143] Not Found: /simulation_tool
[30/Jun/2016 14:55:39] WARNING [django.request:143] Not Found: /Simulation_tool
[30/Jun/2016 15:53:46] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd75d72ba10>.
[30/Jun/2016 15:53:50] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd755f6c250>.
[30/Jun/2016 15:54:07] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7543e5810>.
[30/Jun/2016 15:54:10] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd75d742250>.
[30/Jun/2016 15:54:13] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd75d742a90>.
[30/Jun/2016 15:54:15] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd75cb58c90>.
[30/Jun/2016 15:54:21] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd78ce05c90>.
[30/Jun/2016 16:01:35] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd738fb8b50>.
[30/Jun/2016 16:02:19] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1344, in can_remove
    form_data = FormData.objects.get(id=form_data_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormData matching query does not exist.
[30/Jun/2016 16:02:21] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1344, in can_remove
    form_data = FormData.objects.get(id=form_data_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormData matching query does not exist.
[30/Jun/2016 16:02:23] ERROR [django.request:231] Internal Server Error: /curate/can_remove
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1344, in can_remove
    form_data = FormData.objects.get(id=form_data_id)
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormData matching query does not exist.
[30/Jun/2016 16:02:26] ERROR [django.request:231] Internal Server Error: /curate/generate_xsd_form
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 2051, in generate_xsd_form
    formString += generateForm(request)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1935, in generateForm
    form_data = FormData.objects.get(pk=ObjectId(form_data_id))
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/mongoengine/queryset/base.py", line 186, in get
    raise queryset._document.DoesNotExist(msg)
DoesNotExist: FormData matching query does not exist.
[30/Jun/2016 16:14:12] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7449cc510>.
[30/Jun/2016 16:15:06] DEBUG [oauthlib.oauth2.rfc6749.endpoints.resource:74] Dispatching token_type Bearer request to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7fd7443fc3d0>.
[30/Jun/2016 16:52:16] ERROR [django.request:231] Internal Server Error: /modules/diffusion/upload-excel
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/modules/diffusion/views.py", line 13, in upload_excel_view
    return ExcelUploaderModule().render(request)
  File "/home/NANOMINE/Develop/mdcs/modules/models.py", line 37, in render
    return self._post(request)
  File "/home/NANOMINE/Develop/mdcs/modules/models.py", line 82, in _post
    raise ModuleError('Something went wrong during module update: ' + e.message)
ModuleError: 'Something went wrong during module update: Data not properly sent to server. Please set "file" in POST data.'
[30/Jun/2016 17:24:49] ERROR [django.request:231] Internal Server Error: /curate/generate_xsd_form
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 2051, in generate_xsd_form
    formString += generateForm(request)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1948, in generateForm
    edit_data_tree = etree.XML(str(form_data.xml_data))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2018' in position 1554: ordinal not in range(128)
[30/Jun/2016 17:25:13] ERROR [django.request:231] Internal Server Error: /curate/generate_xsd_form
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 2051, in generate_xsd_form
    formString += generateForm(request)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1948, in generateForm
    edit_data_tree = etree.XML(str(form_data.xml_data))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2018' in position 1554: ordinal not in range(128)
[30/Jun/2016 17:25:39] ERROR [django.request:231] Internal Server Error: /curate/generate_xsd_form
Traceback (most recent call last):
  File "/home/NANOMINE/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 2051, in generate_xsd_form
    formString += generateForm(request)
  File "/home/NANOMINE/Develop/mdcs/curate/ajax.py", line 1948, in generateForm
    edit_data_tree = etree.XML(str(form_data.xml_data))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2018' in position 1554: ordinal not in range(128)
